runtime/pkg/observability/observability.go

```diff
@@ -25,6 +25,7 @@ type Exporter string
 const (
 	NoopExporter       Exporter = ""
 	OtelExporter       Exporter = "otel"
+	FileBasedExporter  Exporter = "file"
```

Comment:
nit: can omit the "based": `FileExporter`

Reply:
Ah I see – on a quick glance, the `FileExporter` type isn't used outside the package, so since its an implementation detail, could it be named `fileExporter` (lowercase) or `fileExporterImpl` instead? So that we can keep the naming of the external constants consistent.